gm4_holographic_tags/data/holographic_tags/functions/create_hologram.mcfunction

```diff
@@ -0,0 +1,12 @@
+# @s = item frame with renamed name tag
+# run from main
+
+summon area_effect_cloud ~ ~ ~ {CustomNameVisible:1,Particle:"block air",Age:-2147483647,Duration:2147483647,Tags:["gm4_hologram"]}
+data modify entity @e[type=area_effect_cloud,tag=gm4_hologram,limit=1,sort=nearest,distance=..0.1] CustomName set from entity @s Item.tag.display.Name
+data modify entity @e[type=area_effect_cloud,tag=gm4_hologram,limit=1,sort=nearest,distance=..0.1] Rotation set from entity @s Rotation
```

Comment:
Why are you copying rotation onto the area effect cloud? Afaict you aren't storing that back onto the item when destroying the hologram so I don't see the point.

Reply:
It's used to detect which item frame to check when putting the name tag back into an item frame. Unlike Sethbling's there can be multiple holograms in a single block, and removing them requires the item frame to be placed on the same block face that the original item frame was placed on to properly be detected.

Reply:
Ah, thanks for the clarification 👌